README.md

```diff
@@ -17,60 +16,30 @@ npm install --save @daostack/client
 ```
 now you can do:
 ```
-import Arc from '@daostack/client'
+import { Arc } from '@daostack/client'
 
 // create an Arc instance
 const arc = new Arc({
   graphqlHttpProvider: "https://subgraph.daostack.io/subgraphs/name/v23",
```

Comment:
Would love for there to be an endpoint that's always pointing to latest. i.e. "https://subgraph.daostack.io/subgraphs/name/latest" or ".../name/daostack"

Reply:
Are you sure you want that? Usually, new versions = API changes, so anything pointing to "latest" would break at any moment. In any case - this discussion would be for an issue in the subgraph repo :-)

Reply:
Very true, I guess that isn't desired. I just remember having an issue where DAOcomponents could no longer access a previous version's endpoint (v20 for example).

Reply:
Do you think the client project should export a subgraph version that it works with? For example: v23.